source/miscellaneous.rst

```diff
@@ -38,9 +38,9 @@ Steps for creating and executing a new jdk jar file
         b. item BatchNeo4jProviderTest
         c. item ReadOperationsStreamTest
         d. item WriteOperationsStreamTest
-Go to ``Bharat-sim Master\\src\\test\\scala\\com.bharatsim.engine\\graph\\neo4j`` and comment out (``ctrl + \``) all the lines in the test files for tests (a) and (b). Open the ``queryBatching`` folder in the same directory and comment out all lines for tests (c) and (d). 
+Go to ``C:\Users\prath\Documents\BharatSim\src\test\scala\com\bharatsim\engine\graph\neo4j`` and comment out (``ctrl + \``) all the lines in the test files for tests (a) and (b). Open the ``queryBatching`` folder in the same directory and comment out all lines for tests (c) and (d). 
```

Comment:
Make sure all the paths are relative to the ``BharatSim`` folder. Unless the person reading this has a user account called ``prath``, this isn't much use to them. Also, commenting out is done using the other slash, i.e ``ctrl + /``.